src/main/java/com/ifactory/press/db/solr/search/ScoringParentQParser.java

```diff
@@ -89,6 +89,6 @@ protected Filter getFilter(Query parentList) {
     }
 
     protected Filter createParentFilter(Query parentQ) {
-        return new FixedBitSetCachingWrapperFilter(new QueryWrapperFilter(parentQ));
+        return new QueryWrapperFilter(parentQ); // rfhi was FixedBitSetCachingWrapperFilter
```

Comment:
Are there any implications for this? Did `FixedBitSetCachingWrapperFilter` go away?